### ISSUE TITLE
test accuracy

### ISSUE THREAD

**Issue by ggyybb** [michuanhaohao/reid-strong-baseline#146]
我在veri上训练了40个epoch，训练时准确率有99以上，如下图
![图片](https://user-images.githubusercontent.com/61568153/80275463-f0111080-8713-11ea-909f-db30ca6fab4d.png)
但是把模型拿出来用test.py测试的时候，只有这么点准确率，如下图
![图片](https://user-images.githubusercontent.com/61568153/80275490-1cc52800-8714-11ea-9bf5-9955c03a2ade.png)
我想问一下怎么办，是过拟合了么，怎么修改呢，实际两张图片测试结果也很不理想。
还有一个问题，下面这四个输出各有什么作用，我用的是100m那个测试的，结果很差，我想应该是过拟合了，但是不知道如何修改，麻烦看一下，嘻嘻
![图片](https://user-images.githubusercontent.com/61568153/80275514-4c743000-8714-11ea-8405-d16b2eebf95d.png)

**Comment by michuanhaohao** [comment]
猜测你在测试的时候load错了模型，可能load了ImageNet预训练的模型之类的，仔细检查一下代码

**Comment by ggyybb** [comment]
> 猜测你在测试的时候load错了模型，可能load了ImageNet预训练的模型之类的，仔细检查一下代码

python3 tools/test.py --config_file='configs/softmax_triplet_with_center.yml' MODEL.DEVICE_ID "('0')" DATASETS.NAMES "('veri')" TEST.NECK_FEAT "('after')" TEST.FEAT_NORM "('yes')" MODEL.PRETRAIN_CHOICE "('self')" TEST.RE_RANKING "('yes')" TEST.WEIGHT "('OUT/resnet50_ibn_a_model_40.pth')" OUTPUT_DIR "('test_veri')"
这是我的权重和参数，应该没问题的，我又测试了几次，一次比一次低。。

**Comment by michuanhaohao** [comment]
![image](https://user-images.githubusercontent.com/23547900/80331080-a9d5c180-8879-11ea-80a9-ef3aca854a4a.png)
你训练的时候已经测试过模型了，rank1在90多，所以模型没有问题，是你测试使用的脚本出了问题，这个暂时看不出什么东西，你如果会断点调试的话一步一步看看到底出了问题。测试结果在变的话说明你模型参数发生了变化，所以模型很可能处于训练状态。你用给定的train和test的sh文件跑一变，如果没有问题的话建议你在train和test的sh文件上修改你更改的部分，熟悉一下project代码，再慢慢的转化到自己输入脚本。你可以检查一下你测试阶段用的是否是r50的模型还是ibn的模型之类的，确保每个参数都和训练时一致的

**Comment by michuanhaohao** [comment]
如果你这是想跑车辆reid的数据集的话，你可以看下这个project，这个ibn50我们已经把rank-1调到96左右的水平了

https://github.com/heshuting555/AICITY2020_DMT_VehicleReID

**Comment by ggyybb** [comment]
> 如果你这是想跑车辆reid的数据集的话，你可以看下这个project，这个ibn50我们已经把rank-1调到96左右的水平了
> 
> https://github.com/heshuting555/AICITY2020_DMT_VehicleReID

ok! 谢谢！我试一下这个新的项目，非常感谢！
